src/main/java/calculator/Calculator.java

```diff
@@ -0,0 +1,24 @@
+package calculator;
+
+public class Calculator {
+
+    long result;
+
+    void addition(long num) {
+        result += num;
+    }
+
+    void subtraction(long num) {
+        result -= num;
+    }
+
+    void multiplication(long num) {
+        result *= num;
+    }
+
+    void division(long num) {
+        if(num == 0) throw new IllegalArgumentException();
+
```

Comment:
한 줄이라도 중괄호는 생략하면 안됩니다. 

```suggestion
    void division(long num) {
        if(num == 0) {
            throw new IllegalArgumentException();
        }

```
적절한 예외 메세지를 작성하는 것이 어떨까요? 로컬에서는 디버깅이 쉽지만 배포되었다면 디버깅이 힘들 것 같아요.